src/api/user/index.js

```diff
@@ -35,6 +36,16 @@ export async function signUp(email) {
   return (await api.call(MUTATION_SIGN_UP, { email })).signUp;
 }
 
+/**
+ * Recover password by email
+ *
+ * @param {string} email - user Email
+ * @return {Promise<boolean>} Response status
+ */
+export async function recoverPassword(email) {
+  return (await api.call(MUTATION_RECOVER_PASSWORD, { email })).resetPassword;
```

Comment:
what is `resetPassword`?

Reply:
This is a mutation in graphql api. To get only `true` response you need to write this. Else you get response like object
```
{
  resetPassword: true
}
```